docs/hub/spaces-sdks-docker-chatui.md

```diff
@@ -0,0 +1,33 @@
+# Chat UI on Spaces
+
+**Hugging Chat** is the interface of Hugging Face enabling everyone to try open-source large language models using Hugging Face's infrastructure. This Docker template enables everyone to deploy their own Hugging Chat based on a model of their choice with a few clicks.
+
+## Deploy your own Chat UI
+
+To get started, simply head [here](https://huggingface.co/new-space?template=huggingchat/chat-ui-template). You can select the hardware to serve the model of your choice.
+
+<a href="https://huggingface.co/new-space?template=huggingchat/chat-ui-template">
+    <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/huggingface.co_spaces_docker_chatui_landing.png" />
+</a>
+
+You should provide a MongoDB endpoint where your chats will be written. Note that Hugging Face does not have access to your chats. Below this, you can select the Hugging Face Hub ID of the model you wish to serve. To provide a database You can also change the generation hyperparameters in the dictionary below. 
```

Comment:
```suggestion
You can **optionally** provide a MongoDB endpoint where your chats will be written. Note that Hugging Face does not have access to your chats. Below this, you can customize your Hugging Chat by selecting the repository ID of the model you wish to serve. You can also change the generation parameters and customize UI elements.
```

Reply:
Might be worth mentioning that with the new persistent storage for spaces, you can just leave the URL blank and still have a persistent DB for your chats!

Reply:
yes, 100%

(feel free to push your changes to the doc directly if @merveenoyan agrees)

Reply:
@nsarrazin you can go ahead :)

Reply:
I addressed this.